cdw2146.md

```diff
@@ -0,0 +1,28 @@
+# Open Source Development
+
+Name: Christopher Webb
+
+UNI: cdw2146
+
+GitHub: [link](https://github.com/chriswebb09)
+
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# Project Proposal
+* Project Idea: Legit(ish) AR flight sim in iOS 
```

Comment:
add link to your project

Reply:
I updated it to an AR project I've been working on. After seeing HW2, I just want to clarify, you mean I should create new one from scratch?